.husky/pre-push

```diff
@@ -2,5 +2,4 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 # yarn test
-yarn build
-yarn validate-ant-changelog
\ No newline at end of file
+yarn build
```

Comment:
why was yarn validate-ant-changelog removed?

Reply:
Because we are going to be shifting toward a different changelog pattern, but will handle adding that in another ticket